buildconfig/CMake/WindowsNSISQt5.cmake

```diff
@@ -8,6 +8,17 @@ set ( QT5_DIST_DLLS
     Qt5PrintSupport.dll
     Qt5Svg.dll
     Qt5OpenGL.dll
+    Qt5Help.dll
+    Qt5Network.dll
+    Qt5Positioning.dll
```

Comment:
Do we need things like Positioning, Qml, QuickWidgets?

Reply:
I haven't seen any errors like that, but I'll build the package on my 18.04 VM and see if I can replicate it. 

I haven't tried without those DLLs, but everything added were things that the dependency walker threw up as missing. But I'll have a play around and see what can be taken out.

Reply:
Okay. If dependency walker says that then its probably correct. I was just suprised it needs to much stuff as we're not using Qml etc.

For the segfault I wonder if the SQL connection is being closed down too late in the python exit process. There is a `shutdown` method on the `MantidHelpWindow`. Perhaps we need to call that in the `MainWindow.closeEvent`?